packages/react/src/components/header/components/navigationLanguageSelector/NavigationLanguageSelector.tsx

```diff
@@ -35,34 +37,48 @@ const LanguageButton = ({ language }) => {
 
 const renderLanguageNode = (language: LanguageOption) => <LanguageButton key={language.value} language={language} />;
 
-export const NavigationLanguageSelector = ({ children, languages }: LanguageSelectorProps) => {
+export const NavigationLanguageSelector = ({
+  children,
+  languages,
+  fullWidthForMobile = false,
```

Comment:
Is there a reason for using a prop for this? 🤔 Couldn't this work much like navigation menu? It's only declared once and the content is delivered via HeaderContext to HeaderActionBarNavigationMenu, which only appears on screens S and XS. `languageSelectorContent` is already in HeaderContext but that seems to be only the extra languages, the `children` in this case.

Reply:
Now `NavigationLanguageSelector` is refactored so, that content of the selector can be defined anywhere on html and it is stored in `languageContext` provider. Content renders automatically to correct place in `ActionBar` or in below `ActionBar` if screen is in S or XS size

Reply:
Excellent!